### ISSUE TITLE
Please add variable $ti-icon-prefix

### ISSUE THREAD

**Issue by geri777** [aastrong/themify-icons.scss#4]
It would be very helpful to be able to determine the prefix of the icon classes.
For example your package uses icon-* whereas Themify.me uses ti-*.

If someone wants to stay with ti-* he could just set `$ti-icon-prefix:"ti-";`
This might help a lot to avoid conflicts when using multiple icon fonts.

Example (_icons.scss):

```
.#{$ti-icon-prefix}user:before {
  content: "\e602";
}
```

**Comment by aastrong** [comment]
Sorry for the crazy late reply. 
Yes I can certainly bake this in, but won't have the time to touch this for a few weeks. 

If you've already done so personally, can you create a Pull Request and I'll Pull it in.

**Comment by aastrong** [comment]
Hey @geri777 - I've created a new branch with your suggestion. 
Mind giving it a test - it defaults to "icon" - but can be changed. 

https://github.com/aastrong/themify-icons.scss/tree/prefix

**Comment by geri777** [comment]
Thank you Aaron! I am not sure if I tested it correctly. But I came across following problems:
1.) I had to edit _themify-icons.scss ... I had to rename all includes and add an underscore. I guess this is due to my lack of knowledge how to build the project.
2.) The CSS which was build from the SCSS still contains loads of icon-* selectors. They come from "_core.scss" and "_mixin.scss" and "_extras.scss"

I have updated this branch with a working copy - but I think more testing is required
